src/saga/diff.js

```diff
@@ -143,8 +77,23 @@ async function diff(context, initialCommit, finalCommit) {
         const initialFormulas = await getFormulas(context, modifiedSheetNamePairs[i].initialSheet);
         const finalFormulas = await getFormulas(context, modifiedSheetNamePairs[i].finalSheet);
 
-        const result = simpleDiff2D(initialFormulas, finalFormulas, modifiedSheetNamePairs[i].sheetName);
-        sheetChanges.push(result);
+        const changes = simpleDiff2D(initialFormulas, finalFormulas, modifiedSheetNamePairs[i].sheetName);
+
+        // No changes, mark it as such
+        if (changes.length === 0) {
+            sheetChanges.push({
+                sheetName: modifiedSheetNamePairs[i].initialSheet, 
+                changeType: changeType.NONE, 
+                changes: []
+            });
+        } else {
+            sheetChanges.push({
+                sheetName: modifiedSheetNamePairs[i].initialSheet, 
```

Comment:
Your returning initialSheet, which is a sheet name the contains a saga-commitID- prefix. Do we want that? You will need to splice it for the UI.